sdk_entrepot_gpf/store/interface/LogsInterface.py

```diff
@@ -58,22 +63,37 @@ def api_logs_filter(  # pylint:disable=too-many-branches
         else:
             i_lastpage = i_total_page
         if i_firstpage > i_lastpage:
-            raise StoreEntityError(f"La dernière page doit être supérieure à la première ({i_firstpage}, {i_lastpage}).")
-
+            raise StoreEntityError(f"La dernière page doit être superieur a la première ({i_firstpage}, {i_lastpage})")
```

Comment:
Orthographe  (mettre à jour dans les tests)
```suggestion
            raise StoreEntityError(f"La dernière page doit être supérieur à la première ({i_firstpage}, {i_lastpage})")
```